mobify/progressive-web-scaffold
# Update checkout shipping for Architecture 2.0

This PR updates the checkout shipping template to the new architecture, and adds missing functionality to the page

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1015
 **Linked PRs**: n/a

## To Do
- [x] Refactor Checkout shipping sections into partials (#291)
- [x] Connect checkout shipping partials to the redux store (#293)
- [x] Parse content from the desktop site for the UI (#298)
- [x] Address design feedback (https://mobify.atlassian.net/browse/WEB-1099) (#308)
- [x] Add missing functionality to checkout shipping page (ex. recognize known email and show sign in form) (#319)
- [x] Add selectors to access the store for all applicable checkout shipping props (#298)
- [x] Persist shipping form data through the API (#305)
- [x] Refetch shipping methods on county or region change (#305)
- [x] Add initial values for shipping form (#314)

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- add an item to the cart
- navigate to: https://www.merlinspotions.com/checkout/
- the page should work as expected and defined in the invision mocks (see: https://mobify.atlassian.net/browse/WEB-1015
